tmp/keyring

```diff
@@ -0,0 +1,3 @@
+[mon.]
```

Comment:
@lxbsz are this and kv_backend related?

Reply:
Sorry, I will remove the tmp files, which are added by mistakes.